west.yml

```diff
@@ -47,6 +47,12 @@ manifest:
     - name: hal_silabs
       revision: 9151e614c23997074acd1096a3e8a9e5c255d5b9
       path: modules/hal/silabs
+    - name: hal_st
+      revision: b6c9262eed68000e69c92bef6e820cdbd5b0a32b
+      path: modules/lib/st
```

Comment:
Can we call it modules/hal/st ?
In here I'm planning to add also STdC HAL i/f to ST sensors.